UofS-Pulse-Binfo/analyzedphenotypes
# Input File now requires method name and unit.

## Metadata

 - Issue #59 




Documentation:

## Description
Since the upload file format only has a column for trait name, if the same trait is in the file with multiple measurements, they will be combined. For example, if we have a time course for canopy height (same trait, multiple methods) then the describe form will only show a single fieldset for canopy height and you will only be able to specify a single method.

Solution: add unique short measurement name and unit to file format and autofill fields based on them.

## Testing?




To test this PR, try uploading files with more then one method per trait. When done you should see both methods listed under the same trait in the trait distribution plot selection form.

Things to keep an eye on:
- check file validation (germplasm index changed)
- check summary statistics in describe stage
- now the method description and unit should be filled in automatically if the trait was already saved with the same method (no more markup with lists to copy/paste from)
- you should not be able to change the unit, method or trait name.
